tests/scripts/all.sh

```diff
@@ -2555,6 +2559,9 @@ component_test_psa_crypto_config_accel_all_curves_except_p192 () {
     loc_accel_list="$loc_accel_list ECC_SECP_K1_224"
     loc_accel_list="$loc_accel_list ECC_SECP_R1_192"
     loc_accel_list="$loc_accel_list ECC_SECP_K1_192"
+    # Just leave SW implementation for the curve with the smallest bit size
+    # (MBEDTLS_ECP_DP_SECP192R1_ENABLED) for allowing to build with ECP_C.
```

Comment:
I think this comment needs updating now that the curve is passed as an argument to this function.